packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/permissions/UsersController.java

```diff
@@ -83,7 +86,10 @@ public ResponseEntity<Void> addRoleToUser(
 				publicGroup.setPermissionRelationships(who, Schema.Relationship.MEMBER.toString());
 			}
 			return reBACService.addRoleToUser(roleName, userId);
-		} catch (final Exception | RelationshipAlreadyExistsException e) {
+		} catch (final RelationshipAlreadyExistsException e) {
+			return ResponseEntity.status(304).build(); // Not Modified
```

Comment:
We could also return `HttpStatus.NOT_MODIFIED` here to avoid the `304` hard coded

Reply:
I had tried but failed to find it - but I guess I was using the wrong HttpStatus (apparently there are 3 that come up immediately... ugh)